### PR TITLE
Added tflint testing

### PR DESCRIPTION
**Pull request by kobrikx**
### ***List of changes:***

   Added TFLint testing and remake Checkov call in ICMK.
### ***How to use:***
 - use `Makefile` in ```/examples/simple``` folder
 - ```make terraform.checkov``` - to test infrastructure with Checkov
 - ```make terraform.tflint``` - to test infrastructure with TFLint
 - ```make terraform.test``` - to test infrastructure with Checkov & TFLint
 

###  ***Testing Done:***

 ```make terraform.checkov```
[asciicast](https://asciinema.org/a/URz2VuVP1X1424FYMUfTtYKhp)

 ```make terraform.tflint```
[asciicast](https://asciinema.org/a/CCW8s3eq6GBr6ymVqPsjoYyNy)

```make terraform.test```
[asciicast](https://asciinema.org/a/oZ0sYjBvCv9fZPs2MHxCiAmwJ)

### CODE REVIEW

File: terraform/main.mk [hunk 1]
```diff
@@ -36,29 +39,32 @@ terraform.init: gomplate terraform
...
 terraform.apply: terraform.init ## Deploy infrastructure
-	@ cd $(INFRA_DIR)/env/$(ENV) && \
+	@ cd $(ENV_DIR) && \
 	$(TERRAFORM) plan -out=tfplan -input=false && \
 	$(TERRAFORM) apply -input=false tfplan && \
 	$(TERRAFORM) output -json > output.json
 
-terraform.test: terraform.init ## Test infrastructure
+terraform.checkov: ## Test infrastructure with checkov
 	$(CHECKOV)
```
Comment (AutomationD) [inline terraform/main.mk:48]: What does this do? Why do we need two separate commands?

File: terraform/main.mk [hunk 2]
```diff
@@ -36,29 +39,32 @@ terraform.init: gomplate terraform
...
-	@ cd $(INFRA_DIR)/env/$(ENV) && \
-	$(TERRAFORM) validate ./ && \
-	$(TERRAFORM) plan -input=false
+	@ cd $(ENV_DIR)
 	@ $(CHECKOV)
 
+terraform.tflint:  ## Test infrastructure with tflint
+	$(TFLINT)
```
Comment (AutomationD) [inline terraform/main.mk:53]: Why do we need two TFLINTs here?

File: terraform/main.mk (hunk 1 above)
Comment (kobrikx) [reply terraform/main.mk]: Fixed. Removed duplicate command.

File: terraform/main.mk (hunk 2 above)
Comment (kobrikx) [reply terraform/main.mk]: Fixed. Removed duplicate command.

**Comment by AutomationD** [comment]
LGTM

**Pull request merged**